docs/en/cloud/bestpractices/asyncinserts.md

```diff
@@ -14,7 +14,14 @@ This is the default behavior when the async_insert setting is set to its default
 
 ![compression block diagram](images/async-01.png)
 
-By setting async_insert to 1, ClickHouse first stores the incoming inserts into an in-memory buffer before flushing them regularly to disk. This asynchronous behavior allows ClickHouse to automatically batch your data up to 100KB (configurable via [async_insert_max_data_size](/docs/en/operations/settings/settings/#async-insert-max-data-size)) or wait for 1 second (since the first insert) (configurable via [async_insert_busy_timeout_ms](/docs/en/operations/settings/settings/#async-insert-max-data-size)) before writing the data to a new part in the object storage.
+By setting async_insert to 1, ClickHouse first stores the incoming inserts into an in-memory buffer before flushing them regularly to disk.
+
+There are three possible conditions that can cause ClickHouse to flush the buffer to disk:
+- buffer size has reached N KB in size (N is configurable via [async_insert_max_data_size](/docs/en/operations/settings/settings.md/#async-insert-max-data-size))
+- at least N second(s) has passed since the last buffer flush (N is configurable via [async_insert_busy_timeout_ms](/docs/en/operations/settings/settings.md/#async-insert-max-data-size))
```

Comment:
Wrong #async-insert-max-data-size instead of #async-insert-busy-timeout-ms

Reply:
Is this resolved yet?

Reply:
@Slach what do you propose?

Reply:
Row 21 needed to be adjusted I guess from:

```
 at least N second(s) has passed since the last buffer flush (N is configurable via [async_insert_busy_timeout_ms](/docs/en/operations/settings/settings.md/#async-insert-max-data-size))
```

to 

```
- at least N second(s) has passed since the last buffer flush (N is configurable via [async_insert_busy_timeout_ms](/docs/en/operations/settings/settings.md/#async-insert-busy-timeout-ms))
```


 Pushed changes 



https://github.com/ClickHouse/clickhouse-docs/pull/898/files#diff-3c6d78ad5c5f83764d1299cf2278eb418c696ed765e86747e3dbdf0e7fb6b685R21